package.json

```diff
@@ -102,14 +102,14 @@
     "react-native-actionsheet": "2.4.2",
     "react-native-image-zoom-viewer": "3.0.1",
     "seamless-immutable": "7.1.4",
-    "stream-chat": "2.6.0",
     "styled-components": "5.2.0",
     "uuid": "8.3.1"
   },
   "peerDependencies": {
     "react": "^15.0.0 || ^16.0.0",
     "react-dom": "^15.0.0 || ^16.0.0",
-    "react-native": "^0.60.0"
+    "react-native": "^0.60.0",
+    "stream-chat": "2.7.x"
```

Comment:
```suggestion
    "stream-chat": "^2.7.0"
```

Reply:
> why are you moving it to a peerDependency?

isn't the usage the same as react components? we don't specifically import it and the user supposed to install the `stream-chat` and pass it down to the components

Reply:
IMO better to lock into patch upgrade only, minor updates can be breaking too sometimes if we miss something